compiler/rustc_trait_selection/src/traits/mod.rs

```diff
@@ -216,7 +216,7 @@ fn do_normalize_predicates<'tcx>(
     // the normalized predicates.
     let errors = infcx.resolve_regions(&outlives_env);
     if !errors.is_empty() {
-        tcx.dcx().span_delayed_bug(
+        tcx.dcx().span_bug(
```

Comment:
likely reachable, though feel free to apply the comment from right after that in this function: 

```rust
// @lcnr: Let's still ICE here for now. I want a test case
// for that.
```